src/app/shared/components/reject-modal-window/reject-modal-window.component.html

```diff
@@ -0,0 +1,27 @@
+<div fxLayout="column" fxLayoutAlign="center center">
+  <h2 mat-dialog-title>{{ modalTitle }}</h2>
+  <p class="confirmation"> {{ modalDescription }} 
+  </p>
+
+  <form [formGroup]="ReasonFormGroup" fxLayout="column" class="reject-form">
+  
+    <label class="step-label"> Вкажіть причину відмови <span class="step-required">*</span></label>
+    <mat-form-field appPlaceholderStyling floatLabel="never" appearance="none">
+      <div [ngClass]="(ReasonFormGroup.get('description').invalid && ReasonFormGroup.get('description').touched) ? 'redBorder' : '' " class="wrapper-border">
```

Comment:
First of all, when you do firstly ngClass, then later you add just class, then the first class will be not applied.
So, in this case, this div will have ONLY wrapper-border, but "red-border" will never apply. 
The thing you want to do, is 
```suggestion
      <div [ngClass]="(ReasonFormGroup.get('description').invalid && ReasonFormGroup.get('description').touched) ? 'redBorder wrapper-border' : '' wrapper-border">
```
Secondly, I truly believe that red borders are already handled by validation.scss. You need to apply it not for the div, but for the textarea. I think you should reconsider the templated, get rid of this div-wrapper and apply classes directly to textarea.  Take a look, how textarea are implemented in other components.

Reply:
Thank you for the explanation. regarding wrapper div, before implement current aproach I just followed other components example, they have wrapper div. However, I realized that the border doesn't appear when textarea is empty. So I checked the validation form scss file and got 'redBoder' class. I thought it exist for cases like my)
Now I've removed wrapper div and added one more class for the textarea. So it works perfectly now) Please confirm if we can leave this one without wrapper div. Thanks again

Reply:
I think now it looks good! If it works correct, then good job:)